commands/microservice

```diff
@@ -5,63 +5,155 @@ help() {
 
 ----------------------------------------------------------------------------------------------------------
 HELP MESSAGE:
-    Usage: c8y bpl $(basename "$0") <commands> --id <microservice-id>
+    Usage: c8y bpl $(basename "$0") <commands> --id <microservice-id> [--tenant <tenant-id>] [--ignoreAlreadyDone]
     Commands:
-      subscribe           Subscribe tenant to microservice
-      unsubscribe         Unsubscribe tenant to microservice
-      help                Show this help message and exit
+      subscribe                             Subscribe tenant to microservice
+      unsubscribe                           Unsubscribe tenant to microservice
+      help                                  Show this help message and exit
     Arguments:
-      -i, --id      ID    The ID of the application/microservice (required)
-      -h, --help          Show this help message and exit
+      -i, --id                ID            The ID of the application/microservice (required)
+      -t, --tenant            TENANT_ID     The ID of the tenant
+      -h, --help                            Show this help message and exit
+      --ignoreAlreadyDone                   For subscription, ignore 409 already subscribed response
```

Comment:
- this should also work for the "unsubscribe" command
- I'm not sure about the naming; maybe "--skipIfAlreadyDone" sounds better?

Reply:
The problem is in an unsubscribe situation, the "already unsubscribed" and the "application is not available" return the same 404 error so I can not differentiate between them. Also, I can not get the response message in any way to check the issue. That's why, I must skip the use of --ignoreAlreadyDone for unsubscription

Reply:
so maybe we should explicitly check if the app exists and is subscribed before executing the "disable" command?